sdk/loadtestservice/azure-developer-loadtesting/README.md

```diff
@@ -0,0 +1,207 @@
+# Azure Developer LoadTesting client library for Java
+
+Azure Developer LoadTesting client library for Java.
+
+This package contains Microsoft Azure Developer LoadTesting client library.
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][docs]
+- [Product documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-developer-loadtesting;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-developer-loadtesting</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authentication
+
+[Azure Identity][azure_identity] package provides the default implementation for authenticating the client.
+
+By default, Azure Active Directory token authentication depends on correct configure of following environment variables.
+
+- `AZURE_CLIENT_ID` for Azure client ID.
+- `AZURE_TENANT_ID` for Azure tenant ID.
+- `AZURE_CLIENT_SECRET` or `AZURE_CLIENT_CERTIFICATE_PATH` for client secret or client certificate.
+
+In addition, Azure subscription ID can be configured via environment variable `AZURE_SUBSCRIPTION_ID`.
+
+With above configuration, `azure` client can be authenticated by following code:
+
+```java
```

Comment:
This sample is not correct. `AzureProfile` is from azure-core-management. User would not use this lib in data-plane.

Please remove it, as well as the line `    .authorityHost(profile.getEnvironment().getActiveDirectoryEndpoint())` (default host should work for Azure Global).

As Sameeksha mentioned, you'd better to put it to a file and use tag to insert snippet (this would let you avoid this issue mentioned above).